index.html

```diff
@@ -0,0 +1,125 @@
+</head>
+<body >
+    <div class="header-container">
+        <div class="slogan-container">
+            <img src="icon/logo.png" class="header-logo" alt="logo agentie"> 
+            <h2 class="slogan">Visează, Explorează, <br> Descoperă</h2>
+        </div>
+        <a class="contact-link" href="contact.html">
+            <img src="icon/telephone.png" class="contact-icon" alt="pictograma telefon">
+            <img src="icon/mail.png" class="contact-icon" alt="pictograma mail">
+        </a>
+    </div>
+    <div class="background-image-container">
+        <nav class="navigation-menu">
+            <div class="menu-items">
+                <a href="index.html" class="menu-button">Acasă</a>
+                <a href="destinations.html" class="menu-button">Destinații</a>
+                <a href="services.html" class="menu-button">Servicii</a>
+                <a href="aboutus.html" class="menu-button">Despre Noi</a>
+                <a href="contact.html" class="menu-button">Contact</a>
+            </div>
+        </nav>
+    </div> 
+    <div class="icon-container">
+        <div class="brief-content">
+            <img src="icon/travel.png" class="brief-content-icon" alt="pictograma planeta">
+            <h4>200+ <br> Destinații</h4>
+            <p class="gray-paragraph">Cea mai mare varietate de destinații. Pentru orice gust și portmoneu</p>
+        </div>
+        <div class="brief-content">
+            <img src="icon/money.png" class="brief-content-icon" alt="pictograma bani">
+            <h4>Cel mai bun <br> preț</h4>
+            <p class="gray-paragraph">Oferim cele mai accesibile prețuri de pe piață</p>
+        </div>
+        <div class="brief-content">
+            <img src="icon/like.png" class="brief-content-icon" alt="pictograma like">
+            <h4>10.000+ <br> Aprecieri</h4>
+            <p class="gray-paragraph">Cea mai populară Agenție de Turism din Romania</p>
+        </div>
+        <div class="brief-content">
+            <img src="icon/rating.png" class="brief-content-icon" alt="pictograma clienti">
+            <h4>8.000+ <br> Clienți</h4>
+            <p class="gray-paragraph">8.000 clienți care au rămas satisfăcuți de serviciile oferite de noi.</p>
+        </div>   
+    </div>
+    <h2 class="top-destinations-title">Destinații de top 2023 <span class="emoji">&#9969;</span> </h2>
+    <div class="destinations-container"> 
+        <div class="img-overlay">
+            <img src="img/dubai.jpg" class="img-destinations" alt="imagine dubai">
+            <div class="overlay">
+                Dubai <br> <span class="price-font">prețuri de la </span> <br>  700 € / pers. 
+            </div>
+        </div>
+        <div class="img-overlay">
+            <img src="img/egipt.jpg" class="img-destinations"" alt="imagine egipt">
```

Comment:
Ghilimele în plus aici

```suggestion
            <img src="img/egipt.jpg" class="img-destinations" alt="imagine egipt">
```